backend/SpareHub/Repository/Neo4J/OrderNeo4jRepository.cs

```diff
@@ -273,8 +380,26 @@ public async Task<List<string>> GetAllOrderStatusesAsync()
     }
 
 
-    public Task<List<Order>> GetOrdersByIdsAsync(List<string> orderIds)
+    public async Task<List<Order>> GetOrdersByIdsAsync(List<string> orderIds)
     {
-        throw new NotImplementedException();
+        const string query = @"
+    MATCH (o:Order)-[:HAS_STATUS]->(s:Status),
+          (o)-[:SUPPLIED_BY]->(sup:Supplier),
+          (o)-[:FOR_VESSEL]->(v:Vessel)-[:OWNED_BY]->(own:Owner),
+          (o)-[:STORED_AT]->(w:Warehouse)
+    OPTIONAL MATCH (o)<-[:BELONGS_TO]-(b:Box)
+    WHERE o.id IN $orderIds
+    RETURN o, s, sup, v, w, own, collect(b) AS boxes";
+
+        var session = driver.AsyncSession();
+        try
+        {
+            var result = await session.RunAsync(query, new { orderIds });
```

Comment:
var result er vel ikke nødvendigt når det ikke bliver brugt. Ellers alt andet ser godt ud!
```suggestion
            await session.RunAsync(query, new { orderIds });
```